src/Microsoft.ML.TimeSeries/STL/LoessBasicParameters.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Microsoft.ML.TimeSeries
+{
+    internal class LoessBasicParameters
+    {
```

Comment:
Are we expecting more members and functionality to come into this class? If not, would it make sense to remove this and just reference this constant directly?
If not, at least merge this class into one of the other relevant files?

Reply:
updated.

---
In reply to: [442527021](https://github.com/dotnet/machinelearning/pull/5202#discussion_r442527021) [](ancestors = 442527021)